### PR TITLE
Cleaning: Make stuff reusable (hedvig-ui)

### PR DESCRIPTION
**Pull request by rasmusguterstam**
## Included

Adds some utilities/components that I've been reusing over and over again, which I believe would be useful in the future as well.

* **`<Badge>`**: A Badge component, simliar to button but even rounder corners and non-clickable
* **`<Capitalized>`**: Make any text within _Capitalized_. Very useful when you have ALL CAPS ENUMS.
* **`<Placeholder>`**: Often there's a scenario where a value is missing or not available. Instead of leaving it empty or using same styling as if there was a value, use a Placeholder component to apply `theme.placeholderColor`: `<Placeholder>Not available</Placeholder>`

For more specifics, check out corresponding story in hedvig-ui!

### CODE REVIEW

File: shared/hedvig-ui/badge.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const variants: ReadonlyArray<NonNullable<BadgeProps['variant']>> = [
+  'danger',
+  'warning',
+  'success',
+  'default',
+]
+
+export const StandardBadge: React.FC = () => {
+  return (
+    <>
```
Comment (vonElfvin) [inline shared/hedvig-ui/badge.stories.tsx:21]:
```suggestion

```

**Review by vonElfvin — DISMISSED** [review]
Beaut 😍

**Comment by palmenhq** [comment]
But really really nice  😍

**Review by vonElfvin — APPROVED** [review]
:)

**Pull request merged**